Update query builder how-to guide (#728)
memgraph/docs: gqlalchemy/how-to-guides/ogm.md

- [**Create indexes**](#create-indexes)
- [**Create constraints**](#create-constraints)

>Hopefully, this guide will teach you how to properly use GQLAlchemy object graph mapper. If you
>have any more questions, join our community and ping us on [Discord](https://discord.gg/memgraph).

:::info

> To test the above features, you need to have [GQLAlchemy installed](/docs/gqlalchemy/installation) and a running Memgraph instance. If you're not sure how to run Memgraph, check out the Memgraph [Quick start](/memgraph/#quick-start).

Reviewer: ```suggestion
To test the above features, you must [install GQLAlchemy](/docs/gqlalchemy/installation) and have a running Memgraph instance. If you're unsure how to run Memgraph, check out the Memgraph [Quick start](/memgraph/#quick-start).
```
